scripts/partition_manager_output.py

```diff
@@ -68,11 +68,13 @@ def add_line(text_before_split, text_after_split):
             add_line(f'{name_upper}_NAME', name)
 
             try:
-                for paf in partition['affiliation']:
-                    ptd = affiliations.get(paf)
-                    if not ptd:
-                        affiliations[paf] = list()
+                # To support single item string for affiliation we convert the string into a list.
+                pafs = partition['affiliation']
+                if isinstance(partition['affiliation'], str):
+                    pafs = [partition['affiliation']]
```

Comment:
could use the variable from above here
```suggestion
                if isinstance(pafs, str):
                    pafs = [pafs]
```

Reply:
That's true